app/views/pokemon/show.html.erb

```diff
@@ -34,8 +34,18 @@
           </dd>
         <% end %>
       </dl>
+      </div>
+      <div class="col l5">
+      <h4>Which Pokemon did you fight?</h4>
+      <%= form_for @pokemon, url: pokemon_increase_evs_path(@pokemon), method: :post do |f| %>
+        <%= f.label :evs %>
+        <%= f.select(:evs, options_from_collection_for_select(@species, "effort_value_id", "name"))%>
+        <%= label_tag "amount" %>
+        <%= number_field_tag "amount", 1, min: 1 %>
```

Comment:
What is the random `1` as the second argument here? Maybe give it a meaningful name?